lib/ocamlorg_web/router.ml

```diff
@@ -8,11 +8,14 @@ let v3_loader root path request =
     else
       Dream.mime_lookup path, path
   in
-  match Asset.read (root ^ path) with
-  | None ->
-    Page_handler.not_found request
-  | Some asset ->
-    Dream.respond ~headers asset
+  let file = Filename.concat root path in
+  Lwt.catch
+    (fun () ->
+      Lwt_io.(with_file ~mode:Input file) (fun channel ->
+          let open Lwt.Syntax in
+          let* content = Lwt_io.read channel in
+          Dream.respond ~headers content))
```

Comment:
Isn't `Dream.from_filesystem` better than constructing a manual response here?

Reply:
We need a custom handler because the requested path is not the path of the file we want to serve (i.e. we need to add an `index.html` if it's not there)

Reply:
(The IO code is taken from `Dream.from_filesystem`)